### ISSUE TITLE
[Search MDN] ...

### ISSUE THREAD

**Issue by yougotwill** [raycast/extensions#12921]
### Extension

https://www.raycast.com/krzysztofzuraw/search-mdn

### Raycast Version

1.76.0

### macOS Version

14.5

#### Error:
```
Error: Cannot parse render tree JSON:
Invalid escape sequence around line 1, column 2696.)
```

### Steps To Reproduce

1. Search MDN for Array.sort
2. Choose Array.prototype.sort

### Current Behaviour

Shows something went wrong screen

### Expected Behaviour

Load documentation about Array.sort

**Comment by j3lte** [comment]
I did a bit of bughunting here, but can't seem to find an error. The markdown that is retrieved seems perfectly valid, but somehow the markdown viewer of `<Detail />` doesn't like it. Unfortunately this is an error that we cannot catch (I tried this with `try {} catch (e) {}`), this is an internal error in the markdown rendering.
I don't know tho whom at Raycast we would have to escalate, they might want to look at their markdown renderer...
